packages/preview-middleware-client/src/adp/quick-actions/fe-v2/registry.ts

```diff
@@ -58,12 +55,4 @@ export default class FEV2QuickActionRegistry extends QuickActionDefinitionRegist
         }
         return definitionGroups;
     }
-
-    protected getComponentContainerFromPage(page: Control): Control | undefined {
-        if (page instanceof XMLView) {
```

Comment:
This is FE V2 specific logic, so this should stay here. We just need to have a version specific branch here for the case when there is no XML view under NavContainer.

Reply:
refactored